ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/services/TimeBasedFragmentCreator.java

```diff
@@ -0,0 +1,49 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.services;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.config.LdesConfig;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.config.ViewConfig;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.entities.FragmentInfo;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.entities.LdesFragment;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.entities.LdesMember;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.entities.TreeRelation;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.repositories.LdesFragmentRespository;
+import org.springframework.stereotype.Component;
+
+import java.util.Optional;
+
+@Component
+public class TimeBasedFragmentCreator implements FragmentCreator {
```

Comment:
I do like this approach immediately ! In the future we might need to extract this to a separate module, but great we already start making an interface!

Reply:
Top!